File: Api_Web/src/test/java/forms/AuthPage.java
package forms;

import aquality.selenium.elements.interfaces.IButton;
import aquality.selenium.elements.interfaces.ITextBox;
import aquality.selenium.forms.Form;
import org.openqa.selenium.By;

public class AuthPage extends Form {
    private final ITextBox loginInput = getElementFactory().getTextBox(By.xpath("//input[@name='login']"), "login input");
    private final IButton submitLogin = getElementFactory().getButton(By.xpath("//div[@class='vkc__Button__title']"),"submit Auth");
    private final ITextBox passwordInput = getElementFactory().getTextBox(By.xpath("//input[@name='password']"), "password input");
    private final IButton submitPassword = getElementFactory().getButton(By.xpath("//div[@class='vkc__Button__title']"),"submit Auth");


    public AuthPage() {
        super(By.xpath("//div[@class='vkc__Auth__pageBox']"),"authPAge");
    }

    public void inputLoginAndSubmit(String login){
        loginInput.clearAndType(login);
        submitLogin.click();
    }

    public void inputPasswordAndSubmit(String password){
        passwordInput.clearAndType(password);
        submitPassword.click();
    }

}

File: Web_API/VkApiTest/VkApiTest/Forms/AuthPage.cs
using Aquality.Selenium.Elements.Interfaces;
using Aquality.Selenium.Forms;
using OpenQA.Selenium;

namespace VkApiTest.Forms
{
    public class AuthPage : Form
    {
        private ITextBox PasswordInput => ElementFactory.GetTextBox(By.XPath("//input[@name='password']"), "password input");
        private IButton ContinueButton => ElementFactory.GetButton(By.XPath("//div[contains(@class,'buttonWrap')]//button"),"Continue button");
        public AuthPage() : base(By.XPath("//input[@name='password']"),"password input") 
        {
        }

        public void InputPassword(string text) => PasswordInput.SendKeys(text);
        public void Continue() => ContinueButton.Click();
    }
}
